components/productions/index/events.vue

```diff
@@ -1,70 +1,78 @@
+            @click="handleClickDelete"
+          >
+            <fa icon="trash" />
+            {{ $t('productions.delete') }}
+          </b-button>
+        </div>
+      </div>
+      <div v-if="isAddEventVisible" class="add-event-container">
+        <input
+          ref="eventIdInput"
+          v-model="eventId"
+          type="text"
+          :class="{
+            'is-invalid': hasAddingError && !(eventId === ''),
+            'form-control': true,
+          }"
+          placeholder="cdbid"
+          @input="handleInputEventId"
+        />
+        <b-button
+          variant="success"
+          :disabled="!eventId"
+          @click="handleClickAddEventToProduction"
+        >
+          <span v-if="!isAdding">
+            <fa icon="check" />
+            {{ $t('productions.confirm') }}
           </span>
-          <a class="add-event-link" @click="handleClickAddEvent">
-            {{ !isAddEventVisible ? $t('productions.create') : '' }}
-          </a>
-          <div v-if="isAddEventVisible" class="add-event-container">
-            <input
-              ref="eventIdInput"
-              v-model="eventId"
-              type="text"
-              :class="{
-                'is-invalid': hasAddingError && !(eventId === ''),
-                'form-control': true,
-              }"
-              placeholder="cdbid"
-              @input="handleInputEventId"
-            />
-            <b-button
-              variant="success"
-              :disabled="!eventId"
-              @click="handleClickAddEventToProduction"
-            >
-              <fa v-if="!isAdding" icon="check" />
-              <loading-spinner v-else class="button-spinner" />
-            </b-button>
-            <b-button
-              variant="danger"
-              @click="handleClickCancelAddEventToProduction"
-            >
-              <fa icon="times" />
-            </b-button>
-          </div>
-        </th>
-      </tr>
-    </thead>
-    <tbody v-if="isTableVisible">
-      <event
-        v-for="event in events"
-        :id="parseEventId(event['@id'])"
-        :key="parseEventId(event['@id'])"
-        :name="event.name[locale] || event.name['nl']"
-        :type="getEventType(event.terms)"
-        :location="event.location.name[locale] || event.location.name['nl']"
-        @clickDelete="handleClickDeleteEvent"
-      />
-    </tbody>
-    <tbody v-else-if="isLoading">
-      <tr>
-        <loading-spinner />
-      </tr>
-    </tbody>
-    <tbody v-else>
-      <tr>
-        <td class="text-center">{{ $t('productions.no_events') }}</td>
-      </tr>
-    </tbody>
-  </table>
+          <loading-spinner v-else class="button-spinner" />
+        </b-button>
+        <b-button
+          variant="danger"
+          @click="handleClickCancelAddEventToProduction"
+        >
+          <fa icon="times" />
+          {{ $t('productions.cancel') }}
+        </b-button>
+      </div>
+      <ul class="list-group">
+        <event
```

Comment:
The root element of `event` is `section`. But semantically each list item should be a `li`.